composer.json

```diff
@@ -39,6 +39,7 @@
         "phpunit/phpunit":  "~5.7.27|^6.0|^7.0",
         "friendsofphp/php-cs-fixer": "^2.2"
     },
+    "target-dir": "Azine/EmailBundle",
```

Comment:
You should not use `target-dir` with psr-4 see https://getcomposer.org/doc/04-schema.md#target-dir

Reply:
ups. Thanks for the info. I thought you removed that by accident. I'll leave it out then.

Reply:
Sorry for the misunderstanding